inference-engine/src/inference_engine/src/ie_network_reader.cpp

```diff
@@ -265,7 +266,7 @@ CNNNetwork details::ReadNetwork(const std::string& modelPath,
 
     if (inputModel) {
         auto ngFunc = FE->convert(inputModel);
-        return CNNNetwork(ngFunc, exts);
+        return CNNNetwork(ngFunc, exts, newAPI);
```

Comment:
can we use `CNNNetwork(std::shared_ptr<ICNNNetwork>)` with ngraph impl and don't change public API for `CNNNetwork`?